ChilledStation: A Bold New Venture for Nanotrasen

EOBGames opened the pull request:
## About The Pull Request
### NOTE: CHILLED IS NOT ENTIRELY FINISHED. THIS PR IS BEING OPENING IN DRAFT TO ALLOW FOR PLAYTESTING OF THE (MOSTLY) COMPLETE STATION LAYOUT VIA TESTMERGE!

Adds a new map: ChilledStation. Planned as an icebox replacement, ChilledStation aims to create a map that _feels_ planetary. Icebox, for all the improvements made to it in the last few years, is ultimately let down by its circumstances- originally designed for space, the bulk of the map has never felt particularly fitting for a planetary setting. New additions obviously help, but the vast majority doesn't quite feel right.

Now, to get away from making snipes at another of our maps and instead getting on to what Chilled brings itself: Chilled is a built-from-scratch map, utilising the icemoon framework. It aims for a mid-to-highpop population band; while the station itself is large, the departments aim for relative compactness and centrally-focused design, ensuring that even at midpop players should be interacting frequently.

As for quality- that's subjective, but I like to think that Chilled brings a unique and fresh style to TG, inspired primarily by the works of Okand and Cogwerks, but obviously divergent in its own way. I'd like to know what people think once they get the opportunity to play it.

I obviously have to thank every mapper we've got, as I think everyone's helped in some form or another with Chilled (after all, it took 3 years from start to finish), as well as special mentions for code assistance to @SpaceSmithers, @LemonInTheDark, and @MrMelbert who all really helped with various code tribbles and troubles. @Maurukas also deserves major kudos for his work on Chilled's various atmos problems.

Full Screenshots (WIP Version):
Ground Level
![chilled_upper](https://user-images.githubusercontent.com/58124831/215164722-319c9a4e-262f-493e-916d-f04aea0d49bd.png)
--------------------------------------------------------------------------------------------------------------------------------------
B1F
![chilled_mid](https://user-images.githubusercontent.com/58124831/215164774-217c88f3-a391-4d12-952b-0e72bd79b001.png)
--------------------------------------------------------------------------------------------------------------------------------------
B2F
![chilled_lower](https://user-images.githubusercontent.com/58124831/215164831-7ad1421e-c87e-46d8-90a5-dd5cc19136a4.png)
--------------------------------------------------------------------------------------------------------------------------------------
B3F (Caves Layer)
![chilled_caves](https://user-images.githubusercontent.com/58124831/215164963-3aba81e6-a8ae-43aa-9542-d73f9268306f.png)
--------------------------------------------------------------------------------------------------------------------------------------
B4F (Depths Layer)
![chilled_depths](https://user-images.githubusercontent.com/58124831/215164929-63e5b664-bac3-4629-a5ba-406ae0ec149c.png)

Currently known issues:

- Visual: The arrivals train travels through space to arrive. This will need resolved in code, and is currently beyond the scope of getting the station finished.
- Gameplay: Maint isn't finished. I'm planning to open this up a bit to collaboration, provided initial testing goes smoothly. Sorry to all the assistant maint-trawling mains- I promise, it'll be in a better state eventually for you.
- Gameplay: Mining surface ruins don't spawn. This is a consequence of the surface being tied to unique effects which I don't want replicated in the cave layer. The areas can be tweaked, but this is pretty minor, affecting (as I recall) only 6 ruins.
## Why It's Good For The Game
As noted above, Icebox has never quite fitted on the icemoon setting. Chilled is built from scratch to serve this purpose, and should be a better showcase of the potential of planetary maps.
## Changelog
:cl: EOBGames, Maurukas
add: ChilledStation has arrived! Look for it in your map vote list at mid-to-highpop, and begin your new venture onto the icemoon today.
/:cl:

EOBGames committed:
chilled's back

EOBGames committed:
playtest prep, part 1

EOBGames committed:
playtest prep, part 2

EOBGames committed:
playtest prep, part 3

also increases to 5 levels

EOBGames committed:
playtest prep, part 4

powergrid powerup

EOBGames committed:
playtest prep, part 5

json alterations to accommodate level 5

EOBGames committed:
permabrig

EOBGames committed:
mining base

EOBGames committed:
mining base

EOBGames committed:
cameras

EOBGames committed:
playtest prep, part 6

EOBGames committed:
playtest prep, part 7?

EOBGames committed:
playtest prep, final

EOBGames committed:
playtest prep, actual final

EOBGames committed:
minor area whoopsie

Time-Green commented:
ITS HAPPENING ITS HAPPENING I LOVE YOU CHILLED HOLY SHIT HAHAHAHAHHAHAHAHA

the-orange-cow commented:
please please please let the power be better than icebox

unit0016 commented:
I have never been as excited to feel completely lost as I am now.

tralezab commented:
Someone call icebox shes crying
AND GOOD ITS GOOD THAT SHE'S CRYING **GET HER OUT OF HERE**

theta891 commented:
having multi layer plasteel walls on armory and SM seems excessive

Jolly-66 commented:
Can you believe it guys
Chilled Station
In just one week
Chilled Station
I'm so happy for this information

EOBGames committed:
linters and TGM

EOBGames committed:
mo linters, mo problems

LemonInTheDark commented:
Oh lord it's 5z he a chonker 
Bully me if this causes clientside lag I have ideas

EOBGames committed:
linters again

EOBGames committed:
minplayers config (sans request)

Cheshify commented:
Incredibly awesome map, I've been following this through development and I couldn't be happier to finally see it.

EOBGames committed:
integrations

EOBGames committed:
multi-z performance

DaydreamIQ commented:
Despite not really needing them Engineering seems to not spawn with any modsuits in their storage containers, their departmental order console is a medical one and I wasn't personally able to find any insulated gloves anywhere

EOBGames commented:
> Despite not really needing them Engineering seems to not spawn with any modsuits in their storage containers, their departmental order console is a medical one and I wasn't personally able to find any insulated gloves anywhere

Taking a look over, these are all problems, yeah- I'd used an industrial suit storage forgetting those don't actually have engi suits in them by default, and must have copied the order console from medbay using the room there as a reference. As for insuls, table stuff is sorta all over the place. I'll sort these all out, and cheers for the reports.

Maurukas commented:
This is so good. I've gotten lost in it for over an hour now, and I feel like I've seen less than ten percent of it. So rather than lavishing you in further praise, have a bunch of minor issues instead.

<details>
<summary> Fixable stuff, some of which might be intentional or just not done yet.</summary>

* Access restrictions on AI upload are missing
* Access on the windoors around the bridge comm console is wrong (currently captain only, should be command unless this is deliberate). Heads will not be able to call the shuttle without breaking in.
* Command quarters (fantastic addition) are missing shutter controls
* Tech storage has lots of empty racks, which looks kind of weird because of how dense the spawned circuit boards are on other racks. More clutter/goodies?
* The SM seems unstable in it's default configuration. I'm not immediately sure why - if it's the multi-z space, but it got unhappy fast and I had to straight pipe everything to stabilize it.
* The SM needs some better meter placement. Try to make sure there's at least one gas meter in each section of the engine loop that's separated by atmos devices to make sure clogs are visually apparent to players.
* Hidden pipes in the SM instead of visible:
![image](https://user-images.githubusercontent.com/66576896/215228231-e150dcdd-352c-4e25-85a6-8d676717c62a.png)
![image](https://user-images.githubusercontent.com/66576896/215228342-58dc58ff-6390-4af4-a2ae-7e40f292e879.png)
* Rear SMES are unreachable over the table:
![image](https://user-images.githubusercontent.com/66576896/215229240-7fb077fd-684e-44cd-ac2f-e2f5a0668478.png)
</details>

If you'd prefer it, I can start tackling things and start PRing to your branch, or just continue to list things here.

EOBGames committed:
playtest fixes, volume one

san7890 commented on _maps/chilledstation.json:
```diff
@@ -0,0 +1,76 @@
+			"Baseturf": "/turf/open/openspace/icemoon/keep_below",
+			"No Parallax": true
+		},
+		{
+			"Down": -1,
+			"Up": 1,
+			"Mining": true,
+			"Linkage": null,
+			"Gravity": true,
+			"Ice Ruins Underground": true,
+			"Baseturf": "/turf/open/openspace/icemoon/keep_below",
+			"No Parallax": true
+		},
+		{
+			"Down": -1,
+			"Up": 1,
+			"Mining": true,
+			"Linkage": null,
+			"Gravity": true,
+			"Ice Ruins Underground": true,
+			"Baseturf": "/turf/open/openspace/icemoon/keep_below",
+			"No Parallax": true
+		},
+		{
+			"Down": -1,
+			"Mining": true,
+			"Linkage": null,
+			"Gravity": true,
+			"Ice Ruins": true,
+			"Weather_Snowstorm": true,
+			"Baseturf": "/turf/open/openspace/icemoon/keep_below",
+			"No Parallax": true
+		}
+	],
+	"minetype": "none",
```
```suggestion
		{
			"Down": -1,
			"Up": 1,
			"Mining": true,
			"Linkage": null,
			"Gravity": true,
			"Baseturf": "/turf/open/openspace/icemoon/keep_below",
			"No Parallax": true
		},
		{
			"Down": -1,
			"Up": 1,
			"Mining": true,
			"Linkage": null,
			"Gravity": true,
			"Baseturf": "/turf/open/openspace/icemoon/keep_below",
			"No Parallax": true
		},
		{
			"Down": -1,
			"Mining": true,
			"Linkage": null,
			"Gravity": true,
			"Weather_Snowstorm": true,
			"Baseturf": "/turf/open/openspace/icemoon/keep_below",
			"No Parallax": true
		}
	],
	"minetype": "none",
```
![image](https://user-images.githubusercontent.com/34697715/215232210-8699b6c3-2f65-495c-84c7-52728d6460ea.png)

The ruin loader is wasting time on every initialize because the z-level is flagged to have these ruins, but you (deliberately) haven't placed any valid areas for it to place stuff into

private-tristan commented:
![Screenshot_20230127_204311_Firefox](https://user-images.githubusercontent.com/54422837/215235242-74875d3b-3e68-4d66-80e8-1cde8a49fc33.jpg)
I think(?) There aren't enough manifolds to properly cool down the supermatter, especially if it gets set on fire also no plasma-lake for manifold cooling :(

Maurukas commented:
<details>
<summary>Heading through medbay now:</summary>

* Morgue chute terminates abruptly here:
![image](https://user-images.githubusercontent.com/66576896/215234246-d54e2fe9-7f8f-4f5a-945b-7206e6b2ea3a.png)
* Medbay's crematorium is set to crematorium access, which medical staff do not have. This should probably be morgue access, as this room is entirely internal to medbay. 
![image](https://user-images.githubusercontent.com/66576896/215234419-b48e008f-93e1-410c-8a55-a785f01ad7c7.png)
* I cannot locate a button for the shutters in pathology on the morgue
* Shipping supplies for the medbay delivery office
* Medbay needs a general lightswitch pass
* Cryo's waste filter is not connected to the atmospheric loop, so cryo is currently nonfunctional. There are also some extra pipes here: 
![image](https://user-images.githubusercontent.com/66576896/215235037-cc710364-6544-4077-a930-0f7abf0bc9be.png)
* This area near the morgue has no lighting (unless this is intentional) 
![image](https://user-images.githubusercontent.com/66576896/215234610-8ce8a15c-31b3-466d-8a39-f6d249648684.png)
* Plumbing APC is not wired
![image](https://user-images.githubusercontent.com/66576896/215235183-aca02f76-803d-4f48-818a-c5eaf86bd93f.png)
* Chemistry is missing some typical roundstart supplies - beakers, pills and other typical clutter.
</details>

ophaq commented:
This is so awesome. Can't wait to play it!

Inari-Whitebear commented:
From playing it last night:
* Some engis said they couldn't find insuls
* Apparently the engi ordering console was the medical one
* Yeah the SM manifolds seemed kinda bad at cooling, used the coolers
* Didn't seem to find an RPD in the SM room or engi areas? Think an atmos tech ended up handing me one, but maybe the CE took the one that was lying around or such

zxaber commented:
Seems pretty *cool*.

<details>
<summary>Anyway, this robotics design with the mech bay and fabs in the same room is really neat.</summary>

![image](https://user-images.githubusercontent.com/37497534/215263527-d7d77529-401f-4f08-b31a-2616888cf0c6.png)
</details>

 Certainly a fresh take on the idea. Though, making the charging area for mechs be the workshop itself makes me wonder if we should have a separate public mech charger, like outside engineering or something.

Also, it's missing borg chargers, and could use a holopad right in the center.

SomeoneYouProbablyKnow commented:
Finally we can axe icebox and have a map that's actually purpose built for planetside

Mey-Ha-Zah commented:
All of my yes. Icebox had potential but I always felt the conversion was half hearted, this looks so good and I LOVE the rafter areas.

EOBGames committed:
updates, volume 1

Jackal-boop commented:
Since this map is an icebox replacement. Will it feature icebox specific mega fauna? Wendigo, frost miner, clock work

EOBGames committed:
areas, updates, volume 2

EOBGames commented:
> Since this map is an icebox replacement. Will it feature icebox specific mega fauna? Wendigo, frost miner, clock work

Yes, everything that spawns on icebox presently should spawn the same here, as it uses the same mechanics. The only current exemption is surface ruins, but I intend to find a fix for those (perhaps moving them to an upper cave level).

> My only concern is the sheer size o this lad. Could be prudent to slap a minimum of like 40 pop on it.

Already set in the map config, and if/when this hits servers more permanently that's already the plan.

> Also, it's missing borg chargers, and could use a holopad right in the center.

There are borg chargers outside in the hallway, although I can see about fitting one or two into robo itself or maybe into an annex across the hall. As for a holopad, felt redundant to have one on the mech floor and one on the surgery floor- I could move it, mind.

Maurukas commented:
Reviewing on the latest commit:

<details>
<summary>Mostly Cargo and related areas</summary>

 * Duplicate APC in the courtroom: (125,98,6) ,(119,85,6) 
 * Roundstart error that Engineering is missing a delivery error
 * Consider additional lighting for the cargo warehouse, also some crate spawns: 
![image](https://user-images.githubusercontent.com/66576896/215289900-d21a4987-408c-4a60-9595-5c857b45019f.png)
* I cannot locate controls for these shutters: 
![image](https://user-images.githubusercontent.com/66576896/215290355-b17c9093-aa9b-4060-a7d6-79015bcbc113.png)
* Cargo elevator still provides access to all of cargo from the mining level. The access restrictions are on the elevator buttons, not the panel, so if the shutters are done anyone can walk in and access the whole department.
* Refrigerator in the mining break is not mining/cargo accessible. Looks like it's the chef's variant: 
![image](https://user-images.githubusercontent.com/66576896/215290579-7b4e5004-e5be-47a6-990e-65ecc68cf1b7.png)
* I cannot locate controls for these shutters with the ATVs: 
![image](https://user-images.githubusercontent.com/66576896/215290769-eaf86885-7889-401d-86cf-ddf4ab0aee4c.png)
* These shutters on the janitorial elevator should be open at roundstart. Pressing the summon elevator button on Z-4 does not open them, requiring the elevator to be summoned to Z-3 before it can be used: 
![image](https://user-images.githubusercontent.com/66576896/215292734-46d9a184-2729-4107-bc04-7d109bbbfbe2.png)
* Same on the science elevator: can't be used from the floor it starts on until it's summoned to another floor once: 
![image](https://user-images.githubusercontent.com/66576896/215293105-09dfad1b-d058-4392-b7c0-8cb96be4d6cb.png)

* The door on the left side of the teleporter has no access set, allowing free access from EVA. I also cannot locate shutter controls: 
![image](https://user-images.githubusercontent.com/66576896/215292831-e86e90a4-a66a-408f-af33-a352df5053a9.png)
* I cannot locate the controls for the EVA shutters
* Missing maint door across from the barbershop:
![image](https://user-images.githubusercontent.com/66576896/215293066-7311500c-9e8c-474e-a76e-4a193c01968e.png)



</details>

BlueMemesauce commented:
The long corriders of this map are very different from the cramped maps tgstation has always had. I think it's more immersive and realistic, but I'm still waiting to see how it plays out gameplay wise. Hopefully things aren't too isolated.

Time-Green commented:
<details>
<summary>Stuff I ran in during playing around on local</summary>
No lights

![image](https://user-images.githubusercontent.com/7501474/215321576-debbaa64-d2e1-4560-9bd1-227c47a7b34f.png)

A lot of the indoor walls are frozen

![image](https://user-images.githubusercontent.com/7501474/215321797-54c36172-258a-489a-910a-4813ca52ef86.png)


Maybe put a warning sign on these external vents? I jumped down in this vent out of curiosity with two others and kinda freaked the atmos tech out how we ended up in 1x1 burn chamber

![image](https://user-images.githubusercontent.com/7501474/215322203-5744a9e5-238f-4a6d-ab56-27830cb10ecf.png)


These chairs are mismatched

![image](https://user-images.githubusercontent.com/7501474/215322289-09f3e34b-1c40-468c-a16d-b6a6b5c47c28.png)

![image](https://user-images.githubusercontent.com/7501474/215322301-ece93ea7-bfd8-4dc0-8674-c9d7b59c2190.png)


</details>

EOBGames committed:
updates, part 3

Maurukas commented:
I tried setting stuff on fire. I found a lot of entertaining things. Some of which could probably use some attention:

<details>
<summary>Fire Safety</summary>

* These corridors could use additional fire alarms. They have only one each and it's some distance from some sets of firelocks: 
![image](https://user-images.githubusercontent.com/66576896/215355148-bb8fd508-3c08-4612-a424-33c21f83a73b.png)
* This is repeated in several other corridors. Chilled has lots of long corridors, many of them have only a single fire alarm. Many of these would benefit from an additional alarm.
* Check staircases, many have fire alarms only on one side: 
![image](https://user-images.githubusercontent.com/66576896/215355432-20a4c383-8b7d-4b3c-8e84-cc65e671fd40.png)
* No firelocks on the courtroom doors
* Most of command has alarms, but no firelocks
* Much of the service area, around the arcade and Jim Nortans have locks but no alarms
* Detective's office has alarms but no locks
* Morgue has alarms but no locks
* Spinward Liason's office has locks but no alarms
* Library reading rooms have locks but no alarms
* 2nd floor of the library  has locks but no alarms (not really a good place for one though)

</details>

<details>
<summary>Atmos resiliency</summary>
An additional couple thoughts since you're working on maintenance now. Large scale breaches to the central area and corridors have pretty devastating effects pretty quickly. This is especially true for vertical punctures through the station's structure. Big chunks of the station get very cold and low pressure very quickly. I recommend adding atmos resiliency when you do maint - spare oxygen and breathing masks, jackets, scrubbers, cans of airmix. Lots and LOTS of space heaters. Maybe even some secondary atmos systems like Kilo has (and doesn't make good use of). Basically - make tools available to the crew to respond to this.
</details>

<details>
<summary>Arrivals tram atmos</summary>
I also recommend some additional holobarriers to the arrivals tram track (like tram has). There's some at one end, but the rest of the track is a single huge interior volume with the mining Z directly below it that runs directly below important parts of the station.
</details>
<details>
<summary>The Botanical Garden</summary>

Finally the arboretum - it contains no vents or scrubbers. I'm not sure if this is intentional, or you just haven't done it yet, as I do see some disconnected pipes in it. This is a huge volume and if someone opens a gas can in there it's going to be impossible to clean up. It might also be cool to stash these in the scaffolding in the top most z-level, since you'll be able to scrub the lower level from there.
</details>

LemonInTheDark commented:
Needs more airlocks looks like

san7890 commented:
btw if you're testing on local and providing small snippets of the affected area it would be absolutely great if you could pull up the View Variables on a turf in the area you're referring to, scrolling down to the "x", "y", "z" portion of the VV panel, and including the coords of that area either in the screenshot or in some text under the screenshot- just so we are 100% sure what you are referring to and can ensure that it's fixed before it's merged into production

thanks

EOBGames committed:
changes, part 4

Maurukas committed:
Atmos work (#22)

Atmos work for:
Engineering corridors
Central Primary corridors
Security + brig cells
Dorms + arrivals shuttle
Minor SM cleanup
Disconnected powergrid in atmos Z:2 + meters

EOBGames committed:
maint work

Maurukas committed:
More atmos (#23)

* Turbine work + test

Linked turbine computer to turbine
linked turbine shutters
ordmix airlock controls
replumbed turbine airlock + added devices
additional qol to atmos - lights, meters, spare cans, missing mixer
tweaked turbine plumbingsas

* tcomms glass subtype

EOBGames committed:
maint, part 1

Maurukas committed:
More atmos fixes - 0 Active Turfs (#24)

* Replace cursed beach turfs

/turf/open/water/beach is an insidious trap because it has no initial gas mix set. It exists to be used as a baseturf and the actual gas mix is set on /turf/open/water/beach/biodome

* 0 ATs

Also fixes runtiming airlock sensor

EOBGames committed:
testmerge time?

EOBGames committed:
testmerge time

EOBGames committed:
testmerge time!

EOBGames committed:
testmerge time!!

EOBGames committed:
testmerge time!!!

EOBGames committed:
testmerge fixes, part 1

Addust commented:
you could give departments helicopters that work like whiteship with different fluff and hyperspace, alongside making the map spawn a syndie helicopter instead of the infiltrator with the same gear

YakumoChen commented:
Never liked stations that are 'too huge' and it'll be interesting to see if the sheer size eventually comes to its detriment like so many stations before (cere comes to mind).

I think Icebox is actually in a good spot right now personally, so not looking forward to this.

Loafin34 commented:
the station's snow and park reminded me of a baystation server I played back in the day, good job!

Grunnyyy commented:
If there is no map specific Vault Safe, I'll be very sad

EOBGames committed:
update time

EOBGames committed:
prelayout test

spockye reviewed (commented):
![image](https://user-images.githubusercontent.com/79304582/230596304-f2d3a3f3-53be-4a09-9618-49b314b37745.png)

missing pipes (mining mech bay)

![image](https://user-images.githubusercontent.com/79304582/230596459-700a194c-113e-487d-bcde-c856cae6cc5a.png)

doesn't this need an output? (xenobio)

![image](https://user-images.githubusercontent.com/79304582/230596789-ab0a3f9d-54f5-43cf-9b5a-d006eec886b5.png)

missing pipes (research division)

![image](https://user-images.githubusercontent.com/79304582/230596987-da7abc5c-c854-44c2-b4e7-16a9b31c7076.png)

missing pipes (atmos office)

![image](https://user-images.githubusercontent.com/79304582/230597075-a22bf75c-8d34-44cd-97e7-1e2fccdea1f0.png)

missing pipes (brig) (both sides)

![image](https://user-images.githubusercontent.com/79304582/230597201-62bb5f16-29e6-4e51-9ec5-411bbdaff8bd.png)

missing pipes (ordenance testing)

![image](https://user-images.githubusercontent.com/79304582/230597261-dc0a94e3-8c1a-4ef5-9f5a-31e95d510f64.png)

missing scrubber (upper science division)

![image](https://user-images.githubusercontent.com/79304582/230597327-7df21569-2e06-413b-8625-6ac9e3acad23.png)

is this supposed to be connected? (science mainteannce)

![image](https://user-images.githubusercontent.com/79304582/230597576-f6b626cf-a362-4019-8e84-d2f86fbe68fc.png)

missing cables (mining comms / labour camp sec)

![image](https://user-images.githubusercontent.com/79304582/230597809-73d253ff-cb26-417b-a423-3c1355ffd3fe.png)

missing wires i think ( mining station)

![image](https://user-images.githubusercontent.com/79304582/230597928-53015834-7a47-4813-ae3a-be9d76be7087.png)

missing cables i think (cargo maintenance)

![image](https://user-images.githubusercontent.com/79304582/230598048-c4fe06d0-4444-4092-9fd9-43358399f020.png)

missing cables i think (ai sat antechamber)

![image](https://user-images.githubusercontent.com/79304582/230598131-1d192250-8a95-40e1-a239-eaa617039aa7.png)

wire wackery (central maintenance)

missing cables i think ( botanical garden gantries)

![image](https://user-images.githubusercontent.com/79304582/230598660-45053629-6d67-4a03-85ac-7ecc911ea0ed.png)

unconnected disposals ( library private study ) 

![image](https://user-images.githubusercontent.com/79304582/230598773-ac9cc64e-8938-4e35-a424-7ec4197a14f4.png)

unconnected disposals (cargo office)

![image](https://user-images.githubusercontent.com/79304582/230598859-0cdd968f-b833-4b29-96ba-3b7f1727450d.png)

missing disposals pipe (dormitories)

![image](https://user-images.githubusercontent.com/79304582/230599157-8540cd80-a898-4107-90ad-e35379e869ec.png)

disposals wackery (medbay)

![image](https://user-images.githubusercontent.com/79304582/230599306-6bafb9b7-fde5-421a-9e89-e1fe8ba7d3b7.png)

more disposals wackery ( medbay)

![image](https://user-images.githubusercontent.com/79304582/230599591-5bea14ed-a17f-4935-b5e4-93028fcfe8d9.png)

missing disposals (robotics / smoothie / genetics) ( and coffee shop ) 

![image](https://user-images.githubusercontent.com/79304582/230599927-1bfe94c1-0e3c-49a7-aae9-6f111f25577a.png)

missing disposals pipes (medbay / chapel maintenance) (also a unconnected bin right below)

![image](https://user-images.githubusercontent.com/79304582/230600156-1ce3dad7-d24f-4755-9ae6-1b7c3c342a60.png)

disposals wackery ( psychs / bar / botanical garden / brig control / brig )

![image](https://user-images.githubusercontent.com/79304582/230600356-c3265303-4ed6-41a3-9485-27e549c8a0c3.png)

disposals wackery (  teleporter / eva / gateway / atmos office )

![image](https://user-images.githubusercontent.com/79304582/230600495-08f2bb8d-8409-4c70-a6df-bbcfb3ddf146.png)

missing disposals pipes ( Head of security office ) 

![image](https://user-images.githubusercontent.com/79304582/230600651-44dcf900-9470-4724-98df-ae0e639b6a1f.png)

zone error

![image](https://user-images.githubusercontent.com/79304582/230600825-e9251ee6-827f-4f5a-aa8b-09fbec7e6bed.png)

atmos maints jumpscare


mmmh, snow
mhmhmhmhm snow.... amazing work

EOBGames committed:
maint changes

optimumtact commented:
open a new pr when this is actually ready?
